nexus-sdk/src/main/java/io/nexusrpc/OperationState.java

```diff
@@ -0,0 +1,8 @@
+package io.nexusrpc;
+
+public enum OperationState {
```

Comment:
Docstring?

Reply:
You can copy from Go: https://pkg.go.dev/github.com/nexus-rpc/sdk-go@v0.0.9/nexus#OperationState

Reply:
Docs updated